Source/Other/Data/SPConstants.h

```diff
@@ -763,23 +761,3 @@ static inline __attribute__((always_inline)) void SABenchmark(void (^block)(void
 #endif
 
 inline __attribute__((always_inline)) NSString *safeString(NSString *str) { return str ?: @""; }
-
-#define foreachGetEnumerator(c) \
-    ([c respondsToSelector:@selector(objectEnumerator)] ? \
-     [c objectEnumerator] : \
-     c)
-#define foreacht(type, object, collection) \
-for ( id foreachCollection = collection; \
-      foreachCollection; \
-      foreachCollection = nil ) \
-    for ( id foreachEnum = foreachGetEnumerator(foreachCollection); \
-          foreachEnum; \
-          foreachEnum = nil ) \
-        for ( IMP foreachNext = [foreachEnum methodForSelector:@selector(nextObject)]; \
-              foreachNext; \
-              foreachNext = NULL ) \
-            for ( type object = foreachNext(foreachEnum, @selector(nextObject)); \
-                  object; \
-                  object = foreachNext(foreachEnum, @selector(nextObject)) )
-
-#define foreach(object, collection) foreacht(id, object, (collection))
```

Comment:
Does this removal pose any risk of breaking things?

Reply:
Shouldn't it was unused.

Reply:
Nope, was just for my tests, @Kaspik ~~deleted~~ sorted them.